### PR TITLE
Add  function

### PR DESCRIPTION
**Pull request by pipermerriam**
### What is the problem / feature ?

`env_int` is a logical addition to this toolset.

### How did it get fixed / implemented ?

- Added an `env_int` function.
- Made some internal adjustments to how things were pulled from the environment so that handling of `default` values could be delegated to each individual `env_*` function.

### How can someone test / see it ?

Look at the README.

*Here is a cute animal picture for your troubles...*

unlikely-animal-friendships-l-4nel_t

### CODE REVIEW

File: excavator/utils.py [hunk 1]
```diff
@@ -1,12 +1,23 @@
 import os
 
 
-def env_string(name, default='', required=False):
+class Empty(object):
+    pass
+
+
+empty = Empty()
```
Comment (wlonk) [inline excavator/utils.py]: You're using this as a singleton-ish thing; maybe `Empty`?

Comment (wlonk) [inline excavator/utils.py]: (You'd obvs have to rename the class or whatever.)

**Comment by wlonk** [comment]
One small question, otherwise :+1:.

File: excavator/utils.py (hunk 1 above)
Comment (pipermerriam) [inline excavator/utils.py]: I actually went the inverse way and changed the class to be `empty` and just used the class as the singleton value.

Chose this approach based on what DRF does.  https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/fields.py#L29

Comment (wlonk) [inline excavator/utils.py]: :+1:

**Pull request merged**